getting_started/setup/setup_up_2_board.rst

```diff
@@ -66,9 +66,9 @@ Copy kconfig fragments for SOF development from https://github.com/thesofproject
 
 .. code-block:: bash
 
-   $ make defconfig
-   $ scripts/kconfig/merge_config.sh .config <path>/kconfig/base-defconfig <path>/kconfig/sof-defconfig  <path>/kconfig/hdaudio-codecs-defconfig
-   $ make -j8
+   make defconfig
+   scripts/kconfig/merge_config.sh .config <path>/kconfig/base-defconfig <path>/kconfig/sof-defconfig  <path>/kconfig/hdaudio-codecs-defconfig
```

Comment:
Need to be careful about those angle brackets. Copy and pasting them wouldn't work out very well. OK for now, I guess, though in the long term I think a better solution is to tell the user to set a variable and use the variable in the instructions.

Reply:
@intelkevinputnam I agree --- I think you referenced this solution in #86. I'll work on it and open a PR.

Reply:
No worries. I just like to beat dead horses.